containersQa/testlib.bash

```diff
@@ -129,28 +130,27 @@ function skipIfRhel7OsExecution() {
 # See https://github.com/containers/podman/issues/19556
 #
 function prepareUserSession() {
-  if [[ -z $USERNAME ]] ; then
-    echo "prepareUserSession: No username specified"
-    return
+  if [[ -z $PD_PROVIDER ]]; then
+    echo "prepareUserSession: No PD_PROVIDER set"
+  else
+    echo "$PD_PROVIDER version:"
+    $PD_PROVIDER --version
   fi
-  echo "prepareUserSession for user \"$USERNAME\""
-  loginctl enable-linger $USERNAME 
-  if [[ $? -ne 0 ]]; then
-    echo "prepareUserSession: FAILED. loginctl enable-linger exit status $?"
-    return
+  if [[ -z $USER ]] ; then
+    echo "prepareUserSession: No username specified"
```

Comment:
if there is no user, how can it continue?

Reply:
Yeah, this is `profilactic` :-) I can remove that, of course.

Reply:
Removed in new commit!